.vib/grafana/cypress/cypress/integration/grafana_spec.js

```diff
@@ -138,27 +51,9 @@ it('checks admin settings endpoint', () => {
   cy.get('@adminSettings').should((response) => {
```

Comment:
Prefer the `cy.request().then()` pattern over aliases, as it adds more complexity and differs from the approach we use in the rest of the tests.